Makefile

```diff
@@ -227,3 +227,15 @@ $(patsubst %,build/%.go,$(BENCHMARKS)): build/%.go: %.py $(COMPILER)
 $(BENCHMARK_BINS): build/benchmarks/%_benchmark: build/benchmarks/%.go $(RUNTIME) $(STDLIB)
 	@mkdir -p $(@D)
 	@go build -o $@ $<
+
+# ------------------------------------------------------------------------------
+# Installation
+# ------------------------------------------------------------------------------
+
+install: $(RUNNER_BIN) $(COMPILER) $(RUNTIME) $(STDLIB)
+	install -Dm755 build/bin/grumpc $(DESTDIR)/usr/bin/grumpc
+	install -Dm755 build/bin/grumprun $(DESTDIR)/usr/bin/grumprun
+	install -d $(DESTDIR)/usr/lib/go
+	cp -rv --no-preserve=ownership build/lib $(DESTDIR)/usr/
```

Comment:
Yes, it's possible to find the correct site-path for the specified Python. It requires some fiddling about, but it is doable. Will fix.